user.go

```diff
@@ -28,17 +28,18 @@ type NotificationRule struct {
 // User is a member of a PagerDuty account that has the ability to interact with incidents and other data on the account.
 type User struct {
 	APIObject
-	Name              string
-	Email             string
-	Timezone          string
-	Color             string
-	Role              string
-	AvatarURL         string `json:"avatar_url"`
-	Description       string
+	ID                string `json:"id"`
```

Comment:
@heimweh ID comes from APIObject struct, which we are embedding here. You can omit this attribute